vtysh/vtysh.c

```diff
@@ -4284,6 +4284,7 @@ void vtysh_init_vty(void)
 	install_element(SEGMENT_ROUTING_NODE, &vtysh_quit_sr_cmd);
 	install_element(SEGMENT_ROUTING_NODE, &vtysh_end_all_cmd);
 
+	install_element(CONFIG_NODE, &segment_routing_cmd);
```

Comment:
To follow the common rule, please, move this command to be right after `install_node(&segment_routing_node);`.

Reply:
Thanks to comment!
I've already applied your kind suggestion!
I'll request review-ing from you again when CI passed 🙇‍♀️

Reply:
Thanks. I'll merge once CI passes.